tests/test_implements_satisfy_protocols.py

```diff
@@ -200,6 +200,18 @@ def test_metric_protocol_derived(metric: PydanticMetric) -> None:  # noqa: D
     assert isinstance(metric, RuntimeCheckableMetric)
 
 
+@given(
+    builds(
+        PydanticMetric,
+        type=just(MetricType.CONVERSION),
+        type_params=builds(PydanticMetricTypeParams, metrics=lists(builds(PydanticMetricInput))),
```

Comment:
I was under the impression that `conversion` metrics don't use `metric.type_params.metrics` but instead use the measure inputs specified in `metric.type_params.conversion_type_params`. Is there a reason why we need to provide them for this test?

Reply:
whoops good catch, i forgot to update this. Updated now!